.github/workflows/e2e_test_run.yaml

```diff
@@ -104,11 +103,3 @@ jobs:
       # ~/.local/bin is added to path runner env through in scripts/env.j2
       - name: test check-jsonschema
         run: check-jsonschema --version
-      - name: Test Firewall
```

Comment:
We also probably don't need the step "Fix snap issue in privileged LXD containers"